index.htm

```diff
@@ -22,22 +22,21 @@ <h1 class="h2 pt-3 font-weight-bold">Rewarding Volunteer Distributed Computing</
             <div class="col-12 py-3">
                 <h3 class="h2 font-weight-bold py-3">What is Gridcoin?</h3>
                 <p>
-                    Gridcoin is an open source <a href="https://en.wikipedia.org/wiki/Cryptocurrency">cryptocurrency</a> (Ticker: GRC) which securely rewards <a href="https://en.wikipedia.org/wiki/Volunteer_computing">volunteer computing</a> performed on the <a href="https://boinc.berkeley.edu/">BOINC platform</a>.

```

Comment:
not sure why this was changed?

Reply:
Clarifies that Gridcoin rewards science work since many people seeing Gridcoin for the first time don't know what volunteer computing or BOINC is, and they should still be able to get an understanding from the first sentence